code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -97,6 +97,22 @@ nobiliumsuppression = INFINITY
 			air.gases[/datum/gas/water_vapor][MOLES] -= MOLES_GAS_VISIBLE
 			. = REACTING
 
+//freon: helps the station chill out
+/datum/gas_reaction/freon_freeze
+	priority = 1
+	name = "Freon Freeze"
+	id = "freon_freeze"
+
+/datum/gas_reaction/freon_freeze/init_reqs()
+	min_requirements = list(/datum/gas/freon = MOLES_GAS_VISIBLE)
+
```

Comment:
So whenever there's freon the tile will cool down to space temps, but with a super high heat capacity gas.